sympy/geometry/ellipse.py

```diff
@@ -870,7 +871,7 @@ def normal_lines(self, p, prec=None):
         x, y = Dummy('x', real=True), Dummy('y', real=True)
         eq = self.equation(x, y)
         dydx = idiff(eq, y, x)
-        norm = -1/dydx
+        norm = -1 / dydx
```

Comment:
Please check through the diff and close up space around `/`.

Reply:
@smichr  Done. Please take a look